src/server-dev.ts

```diff
@@ -30,6 +30,11 @@ process.nextTick(async () => {
                 versionCheck: {
                     enable: false,
                 },
+                ui: {
+                    flags: {
+                        S: true,
+                    },
+                },
```

Comment:
Is this needed? I noticed the flag was named SE but this is S, although I haven't spotted it being used. Not a big deal though!

Reply:
oh, no, I just wanted to test locally. I will remove it.